build_routereflector/README.md

```diff
@@ -41,7 +41,7 @@ Run the following command to start the Route Reflector container image.
 docker run -privileged -net=host -d                                \ 
            -e IP=<IPv4_RR>                                         \
            [-e IP6=<IPv6_RR>]                                      \
-           -e ETCD_AUTHORITY=<ETCD_IP:PORT>                        \
+           -e ETCD_ENDPOINTS=<http://NODE:PORT>                    \
```

Comment:
I think I rather change NODE to ETCD_IP since ETCD_IP is used in other parts of README as well.